bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -494,32 +495,50 @@ layout.group_layout(rdir['single_triggers'], snrchi)
 hist_summ = []
 for insp in full_insps:
     outdir = rdir['single_triggers/%s_binned_triggers' % insp.ifo]
-    wf.make_singles_plot(workflow, [insp], hdfbank, censored_veto,
-                         'closed_box', outdir, tags=['full_data'])
+    singles_plots = wf.make_singles_plot(
+        workflow,
+        [insp],
+        hdfbank,
+        censored_veto,
+        'closed_box',
+        outdir,
+        tags=['full_data']
+    )
+    layout.group_layout(outdir, singles_plots)
     # make non-summary hists using the bank file
     # currently, none of these are made
     outdir = rdir['single_triggers/%s_trigger_histograms' % insp.ifo]
     wf.make_single_hist(workflow, insp, censored_veto, 'closed_box', outdir,
                         bank_file=hdfbank, exclude='summ',
                         tags=['full_data'])
+
+
     # make summary hists for all templates together
     # currently, 2 per ifo: snr and newsnr
     outdir = rdir['single_triggers/%s_trigger_histograms' % insp.ifo]
     allhists = wf.make_single_hist(workflow, insp, censored_veto, 'closed_box',
                                    outdir, require='summ', tags=['full_data'])
+    layout.group_layout(outdir, allhists)
+
     # make hists of newsnr split up by parameter
     # currently, 1 per ifo split by template duration
     outdir = rdir['single_triggers/%s_binned_histograms' % insp.ifo]
     binhists = wf.make_binned_hist(workflow, insp, censored_veto,
                                    'closed_box', outdir, hdfbank,
                                    tags=['full_data'])
+    layout.group_layout(outdir, binhists)
     # put raw SNR and binned newsnr hist in summary
     hist_summ += list(layout.grouper([allhists[0], binhists[0]], 2))
 
 if workflow.cp.has_option_tags('workflow-matchedfilter',
                                'plot-throughput', tags=['full_data']):
-    wf.make_throughput_plot(workflow, full_insps, rdir['workflow/throughput'],
-                            tags=['full_data'])
+   throughput_plots = wf.make_throughput_plot(
+        workflow,
+        full_insps,
+        rdir['workflow/throughput'],
+        tags=['full_data']
+    )
+   layout.group_layout(rdir['workflow/throughput'], throughput_plots)
```

Comment:
As of right now, this is missing the injection throughput plots - I will add these

Reply:
Actually, as the plots don't have titles (that should be changed), this is probably for the best